dapr/clients/grpc/client.py

```diff
@@ -339,6 +341,58 @@ def get_state(
             data=response.data,
             headers=call.initial_metadata())
 
+    def get_states(
+            self,
+            store_name: str,
+            keys: Sequence[str],
+            parallelism: int = 1,
+            metadata: Optional[MetadataTuple] = ()) -> DaprResponse:
```

Comment:
What about the metadata passed along with the call? https://github.com/dapr/python-sdk/pull/115/files#diff-1f8d2ea8f8644e8fdb58ad256d41bcc6R339 That is generally what the MetadataTuple is used for.

Reply:
For some reason, it as added to bulk API request. I moved it to behave like the other calls.